--- a/configure.in
+++ b/configure.in
@@ -225,7 +225,7 @@
 
 #-------------------------------------------------------------------------------
 
-LDFLAGS="$CFLAGS"
+#LDFLAGS="$CFLAGS"
 LDFLAGS_BIN="$LDFLAGS"
 
 if test "x$FC" = "xnone" ; then
@@ -362,7 +362,7 @@
         AC_DEFINE(_LARGEFILE_SOURCE)
         AC_DEFINE(_FILE_OFFSET_BITS,64)
         # LDFLAGS used by utilities:
-        LDFLAGS_BIN="$LDFLAGS_BIN -Wl,-rpath,\${CFITSIO_LIB}"
+        #LDFLAGS_BIN="$LDFLAGS_BIN -Wl,-rpath,\${CFITSIO_LIB}"
 	;;
     *mingw32*)
         AC_MSG_CHECKING([for large file support])
