---
 cfitsio.pc.in |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/cfitsio.pc.in
+++ b/cfitsio.pc.in
@@ -7,6 +7,6 @@
 Description: FITS File Subroutine Library
 URL: https://heasarc.gsfc.nasa.gov/fitsio/
 Version: @CFITSIO_MAJOR@.@CFITSIO_MINOR@.@CFITSIO_MICRO@
-Libs: -L${libdir} -lcfitsio
+Libs: -L${libdir} -lcfitsio -lpthread
 Libs.private: -lm @LIBS@
 Cflags: -I${includedir}
