Description: add sunway support
Author: Miao Changwei <miaochangwei@uniontech.com>

---
Last-Update: 2022-05-19

--- cfitsio-4.0.0.orig/f77_wrap.h
+++ cfitsio-4.0.0/f77_wrap.h
@@ -18,7 +18,8 @@
     ||  defined (__x86_64__) \
     ||  defined (_SX) \
     ||  defined (__powerpc64__)\
-    ||  defined (__s390x__)
+    ||  defined (__s390x__) \
+    ||  defined (__sw_64__)
 
 #define   LONG8BYTES_INT4BYTES
 
--- cfitsio-4.0.0.orig/fitsio.h
+++ cfitsio-4.0.0/fitsio.h
@@ -180,7 +180,8 @@ SERVICES PROVIDED HEREUNDER."
     ||  defined(__x86_64__) \
     ||  defined(_SX)        \
     ||  defined(__powerpc64__) || defined(__64BIT__) \
-    ||  (defined(_MIPS_SZLONG) &&  _MIPS_SZLONG == 64)
+    ||  (defined(_MIPS_SZLONG) &&  _MIPS_SZLONG == 64) \
+    ||  defined(__sw_64__)
 /* sizeof(long) = 64 */
 #define LONGLONG_MAX  9223372036854775807L /* max 64-bit integer */
 #define LONGLONG_MIN (-LONGLONG_MAX -1L)   /* min 64-bit integer */
--- cfitsio-4.0.0.orig/fitsio2.h
+++ cfitsio-4.0.0/fitsio2.h
@@ -151,6 +151,13 @@ extern int Fitsio_Pthread_Status;
 #    error "can't handle long size given by __riscv_xlen"
 #  endif
 
+/* Sunway is always little endian */
+
+#elif defined(__sw_64__)
+
+#define BYTESWAPPED TRUE
+#define LONGSIZE 64
+
 /* ============================================================== */
 /*  the following are all 32-bit byteswapped platforms            */
 
